core/src/main/java/io/questdb/griffin/engine/functions/regex/RegexpReplaceStrFunctionFactory.java

```diff
@@ -172,6 +175,14 @@ public boolean isRuntimeConstant() {
         public void toPlan(PlanSink sink) {
             sink.val("regexp_replace(").val(value).val(',').val(pattern).val(',').val(replacement).val(')');
         }
+
+        private boolean find() {
+            try {
+                return matcher.find();
+            } catch (StackOverflowError err) {
```

Comment:
this looks a bit crazy, is this a normal patten of behaviour of regex?

Reply:
It is possible for certain patterns and very long input strings  due to excessive backtracking .